indico/modules/vc/templates/manage_event_select.html

```diff
@@ -1,21 +1,30 @@
-{% extends 'layout/base.html' %}
-
-{% block title %}{% trans %}Video services{% endtrans %}{% endblock %}
-
-{%- block content %}
-    {%- if plugins %}
-        <p>{% trans %}The following video services are available. Click on one to create a new room:{% endtrans %}</p>
-        <div class="i-badges-list-left i-badges-list-hover">
-            {%- for plugin in plugins %}
-                <a class="js-plugin-details i-badge i-badge-gray" href="{{ url_for(vc_room_action, event, service=plugin.service_name) }}">
-                    <div class="i-badge-content">
-                        <span class="i-badge-img">
-                            <img src="{{ plugin.logo_url }}">
-                        </span>
-                        <span class="i-badge-title">{{ plugin.title }}</span>
-                    </div>
-                </a>
-            {%- endfor %}
-        </div>
-    {%- endif %}
-{%- endblock %}
+{%- if plugins %}
+    <p>{%  set action=_("create") if vc_room_action==".manage_vc_rooms_create" else _("attach") %}
```

Comment:
besides the awful inlining (`<p>` is enough in that line) and the double space / missing spaces:

- pass a boolean argument (e.g. `attach=True`) for this instead of relying on the endpoint name
- don't use string concatenation to insert translated words into translated strings. repeat the whole string (using a "full" set of if..else..endif tags) - otherwise it can't be translated properly
- avoid leading/trailing space inside trans tags